diagnostic.md

```diff
@@ -150,7 +175,8 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+the div's one,two,three, and five will be all in a line starting from the right
```

Comment:
```.divOne``` will be in the top-right corner;
```.divTwo``` will be in the top-left corner, with .divThree immediately to its right;
```.divFour``` will be underneath ```.divTwo```, and ```.divFive``` will sit immediately to the right of ```.divFour``` (under ```.divThree```).